services/frontend-v3/src/components/employmentEquity/EmploymentEquityView.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { List, Tag, Row, Empty } from "antd";
+import { FormattedMessage } from "react-intl";
+
+const EmploymentEquityView = ({ groups }) => {
+  if (groups.length > 0) {
+    return (
+      <Row style={{ paddingBottom: "8px", paddingTop: "8px" }}>
```

Comment:
I know this is small but could you add these to an scss file? So that we try to stay consistent